packages/core/src/commands/plugin-install.ts

```diff
@@ -49,12 +49,10 @@ export class Command extends Commands.Command {
         const pkg: string = this.getArgument("package");
 
         try {
-            await this.install(pkg);
+            return await this.install(pkg);
         } catch (error) {
             throw new Error(error.message);
         }
-
-        throw new Error(`The given package [${pkg}] is neither a git nor a npm package.`);
```

Comment:
Could we preserve this error and move it into `this.install()` **after** the `for` loop? Placing it after the `for` loop means that the given pkg couldn't be found on any available source.